operator/cmd/mesh/testdata/manifest-generate/input/gateways-with-custom-tags-and-no-labels.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: install.istio.io/v1alpha1
```

Comment:
This looks like we added the start of a test but never actually wrote a test case utilizing it?

Reply:
oops forgot to commit it